### PR TITLE
[MAINTENANCE] Remove more refs to usage statistics

### PR DESCRIPTION
**Pull request by cdkini**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/tests/conftest.py [hunk 1]
```diff
@@ -127,11 +127,6 @@ def pytest_addoption(parser):
         action="store_true",
         help="If set, execute tests against snowflake",
     )
-    parser.addoption(
-        "--aws-integration",
-        action="store_true",
-        help="If set, run aws integration tests for usage_statistics",
```
Comment (billdirks) [inline contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/tests/conftest.py:133]: This is really all this flag gates? It's such a general time for a usage stats test.

File: contrib/experimental/great_expectations_experimental/tests/conftest.py [hunk 2]
```diff
@@ -69,11 +69,6 @@ def pytest_addoption(parser):
         action="store_true",
         help="If set, execute tests against snowflake",
     )
-    parser.addoption(
-        "--aws-integration",
```
Comment (billdirks) [inline contrib/experimental/great_expectations_experimental/tests/conftest.py:73]: Same question as the other comment.

**Review by billdirks — APPROVED** [review]
This LGTM. That one flag's name makes it sounds like it gates more even though the help statement only mentions usage stats. Maybe double check where it is used?

File: contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/tests/conftest.py (hunk 1 above)
Comment (cdkini) [reply contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/tests/conftest.py:133]: yup a quick grep shows that its no longer used (and was only used for usage stats before)

**Pull request merged**